cli/index.ts

```diff
@@ -144,11 +144,13 @@ async function importCompanies(conn: Connection, logger: Logger) {
       }
     })
     .on('end', async () => {
-      const values = companies.slice(0);
-      companies = [];
-      await insertCompanies(conn, logger, values);
-      await insertIndexCompositions(conn, logger, SP500_INDEX_SYMBOL, values);
-      logger.info(`All companies imported to DB.`);
+      if (companies.length > 0) {
+        const values = companies.slice(0);
+        companies = [];
+        await insertCompanies(conn, logger, values);
+        await insertIndexCompositions(conn, logger, SP500_INDEX_SYMBOL, values);
+        logger.info(`All companies imported to DB.`);
```

Comment:
The `logger.info` should be placed outside the `if` block.

Reply:
And could you pease also add checking logic to `importIndexPriceHistory` and `importStockPriceHistory`? These functions should have same issues

Reply:
Thanks for the review. Add two more commits.